src/schema.cpp

```diff
@@ -88,6 +88,19 @@ void Schema::validate() const
     }
 }
 
+namespace {
+struct IsNotRemoveProperty {
+    bool operator()(SchemaChange sc) const { return sc.visit(*this); }
```

Comment:
Is `SchemaChange` a cheap enough type to copy that it's not worth taking by reference?